fairlearn/reductions/_moments/error_rate.py

```diff
@@ -9,14 +9,57 @@
 from fairlearn._input_validation import _validate_and_reformat_input
 
 
+_MESSAGE_BAD_COSTS = "costs needs to be a dictionary with keys " \
+    "'fp' and 'fn' and non-negative values"
+
+
 class ErrorRate(ClassificationMoment):
-    """Misclassification error."""
+    r"""Misclassification error as a moment.
+
+    A classifier :math:`h(X)` has the misclassification error equal to
+
+    .. math::
+      P[h(X) \ne Y]
+
+    It is also possible to specify costs for false positives and false
+    negatives. The error then evaluates to
+
+    .. math::
+      c_{FP} P[h(X)=1, Y=0] + c_{FN} P[h(X)=0, Y=1]
+
+    where :math:`c_{FP}` and `c_{FN}` are the costs of false positive
+    and false negative errors respectively. The standard misclassification
+    error corresponds to :math:`c_{FP}=c_{FN}=1.0`.
+
+    Parameters
+    ----------
+    costs : dict
+        The dictionary with keys ``"fp"`` and ``"fn"`` containing the
```

Comment:
That was the intention, but I'm not sure. Website didn't recompile when I pushed the documentation commit, so I cannot tell whether it worked. I've seen both in the docs and they both seem to render similar. Do you have a preference between the two?

Reply:
I suppose the code one since it's explicit while you need to know additional things for the double backquote(?)
You should definitely try how it renders locally when submitting doc changes 😉

Reply:
done